dbfit-java/derby/src/main/java/dbfit/environment/DerbyEnvironment.java

```diff
@@ -84,7 +85,80 @@ public Pattern getParameterPattern() {
 
     public Map<String, DbParameterAccessor> getAllProcedureParameters(
             String procName) throws SQLException {
-        throw new UnsupportedOperationException();
+        String[] qualifiers = procName.split("\\.");
+        String schemaName, procedureName;
+        if (qualifiers.length == 2) {
+            schemaName = qualifiers[0];
+            procedureName = qualifiers[1];
+        } else {
+            schemaName = getConnection().getSchema();
+            procedureName = qualifiers[0];
+        }
+        procedureName = procedureName.toUpperCase();
+        DatabaseMetaData dbmd = getConnection().getMetaData();
+        ResultSet rs;
+        rs = dbmd.getProcedures(null, schemaName, procedureName);
```

Comment:
Is it possible to have the same name both as function and as a procedure? (It's good to know if we have such limitation here. I think we don't properly support such cases in other database types either yet).